src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+import java.util.Scanner;
+import duke.task.*;
+import duke.exceptions.*;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+
 public class Duke {
```

Comment:
There are little or no [javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments) across your files. Consider including them to provide a summary of longer methods.